src/IW/App/Error.hs

```diff
@@ -118,7 +123,20 @@ data AppErrorType
     | UrlDownloadFailed Url
     deriving (Show, Eq)
 
--- | Map the @github@ library's @Error@ type into AppErrorType.
+type CabalErrorInfo = (Maybe Version, [CabalPError])
```

Comment:
Let's make it a data type with the record fields instead? What do you think?

Reply:
@vrom911 Looks good to me.